lib/commands/serve.js

```diff
@@ -58,6 +58,20 @@ module.exports = Command.extend({
     { name: 'watcher', type: String, default: 'events', aliases: ['w'] },
     { name: 'live-reload', type: Boolean, default: true, aliases: ['lr'] },
     { name: 'live-reload-host', type: String, aliases: ['lrh'], description: 'Defaults to host' },
+    {
+      name: 'live-reload-js-url',
+      type: String,
+      aliases: ['lrju'],
```

Comment:
Let's remove the alias, I don't think this option is common enough to need an alias.